nvme_gw_server.py

```diff
@@ -35,8 +35,9 @@ def start_spdk(self):
         self.spdk_rpc = spdk_rpc
         spdk_tgt = self.nvme_config.get("config", "spdk_tgt")
         spdk_cmd = os.path.join(spdk_path, spdk_tgt)
+        rpc_socket = self.nvme_config.get("spdk", "rpc_socket")
```

Comment:
Similar comment here - spdk_rpc_socket or spdk_socket seems more appropriate to me

Reply:
ok about this.